### ISSUE TITLE
installation problems

### ISSUE THREAD

**Issue by HASGAmes** [fionafibration/team-frotress-2#6]
Hi i tried my best to follow the instructions and managed to install all the requirements and got the config copied with the path but i run the python file and it just says config not found and i don't know why. I also couldn't find the console.log file i did check in the tf folder and searched it but it wasn't there. Not sure what i am doing wrong. Also it seems to delete the stuff i paste into the config file on launch for some reason.

**Comment by Felinitie** [comment]
just wanting to add to this, i dont know coding of any sorts, im dumb and horny and i was hoping to get this going but its so many things at once i dont even understand the requirements, im new to using github, im so lost and the instructions feel like im supposed to know a good understanding of python/coding/ect and im really hoping that y'all could dumb it down cause im just hopelessly lost on how to do anything. i love the project but goodness i dont know where to start or how to even connect the dots in any proportion. i've asked 3 separate people who "have an understanding of python" and they couldnt help me set this up. im going to ask my gf if it can understand this at all, but if she cant do it idk who i can turn to at this point

**Comment by semper-lux** [comment]
> Hi i tried my best to follow the instructions and managed to install all the requirements and got the config copied with the path but i run the python file and it just says config not found and i don't know why. I also couldn't find the console.log file i did check in the tf folder and searched it but it wasn't there. Not sure what i am doing wrong. Also it seems to delete the stuff i paste into the config file on launch for some reason.

Could you post the path of the config file that is being overwriten?

**Comment by HASGAmes** [comment]
![image](https://github.com/fionafibration/team-frotress-2/assets/71510248/3f92e130-206e-42e3-ba5a-2e9517b6d667)
it was in that config file and it got overwritten on launch
